src/main/java/com/atlassian/bitbucket/jenkins/internal/model/deployment/BitbucketCDCapabilities.java

```diff
@@ -0,0 +1,30 @@
+package com.atlassian.bitbucket.jenkins.internal.model.deployment;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Set;
+
+import static java.util.Collections.unmodifiableSet;
+import static java.util.Objects.requireNonNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketCDCapabilities {
+
+    public static final String DEPLOYMENTS = "deployments";
```

Comment:
just checking that all is okay because we have

`public static final String DEPLOYMENTS_CAPABILITY_KEY = "deployment";` (no s on the end) inside `AtlassianServerCapabilities`.

Reply:
```suggestion
    public static final String DEPLOYMENTS_CAPABILITY = "deployments";
```

Reply:
> just checking that all is okay because we have

Yes, the server capability is called `deployment` and the deployment capability is called `deployments`. Maybe that's something we should change on the Bitbucket side though before it gets released so that it is more consistent